core-connector/src/main/java/com/modusbox/client/processor/PadLoanAccount.java

```diff
@@ -0,0 +1,22 @@
+package com.modusbox.client.processor;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+
+public class PadLoanAccount implements Processor {
```

Comment:
I think we should get rid of this check and return an error to the user if the account is not properly padded, otherwise in Mojaloop HUB the ledger/logs/reports will be using the wrong non-padded version entered by the customer